cmd/kratos/internal/project/project.go

```diff
@@ -39,6 +40,7 @@ func init() {
 	CmdNew.Flags().StringVarP(&branch, "branch", "b", branch, "repo branch")
 	CmdNew.Flags().StringVarP(&timeout, "timeout", "t", timeout, "time out")
 	CmdNew.Flags().BoolVarP(&nomod, "nomod", "", nomod, "retain go mod")
+	CmdNew.Flags().StringVarP(&moduleName, "module-name", "m", moduleName, "module name")
```

Comment:
can use the args[0] as project name or module name.when args[0] is a module name,project name is the final path element of args[0]